### DISCUSSION TITLE
Proxy request through if no matched rules

### DISCUSSION THREAD

**Discussion by JonnyBooker** [mockoon/mockoon#650] [General]
I was using Mockoon fairly recently and was looking to use it with the proxy functionality in our workplace. I came across this bug on in the closed list and would like to add on to this as to register my interest in the feature they were discussing. Rather then commenting on an old bug, thought it better to do a discussion. 

The bug in question is: #278 

I think there are cases where this is useful. For example, you might have a Third Party API where the setup for certain scenarios either isn't possible or is particularly difficult. 

The latest example I have had of this is a payment API. In our test environments, we would like to integrate with our third party, but also it would be nice to able to force scenarios via sending in known data which would force error responses. The third party does offer some failure scenarios but being able to allow some data through and not others would be a very powerful resource to create our own scenarios.

This would even be useful in the case of our own APIs where we would like to force certain data to come through for certain rules and allow others to proxy. It would be a much richer experience being able to do through Mockoon over a custom implementation each time.

Open to more thoughts on the subject and to discuss it further!

**Comment by 255kb** [comment]
Thank you for your feedback @JonnyBooker 
It's true that such feature could be useful. A similar issue has been opened a while ago: #363 
Feel free to upvote it.

**Reply by cchandrakanth** [reply]
Rather we can add a proxy response type and also rules to determine which requests go to proxy. Like request has some specific SSNs

**Reply by 255kb** [reply]
This is exactly what #363 is about.
